libs/single-spa-angular/webpack/index.ts

```diff
@@ -35,16 +35,21 @@ export default (config: any, options?: Options, extraOptions?: DefaultExtraOptio
     externals: ['zone.js'],
     devServer: {
       historyApiFallback: false,
-      contentBase: path.resolve(process.cwd(), 'src'),
+      static: {
+        // The `devServer.contentBase` property isn't used anymore, the Angular CLI sets the `resolve.roots` property
+        // that references the project root directory.
+        // See: https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/webpack/configs/common.ts#L304
+        // Otherwise, we just fallback to `src` folder.
+        directory: Array.isArray(config.resolve?.roots)
+          ? config.resolve.roots[0]
+          : path.join(process.cwd(), 'src'),
+      },
       headers: {
         'Access-Control-Allow-Headers': '*',
       },
     },
     module: {
-      rules: [
-        {
-        },
-      ],
+      rules: [{}],
```

Comment:
The empty object seems like it could be removed, but already was in the code so i guess it's fine